generator/lib/src/field_element_ex.dart

```diff
@@ -185,15 +185,6 @@ extension FieldElementEx on FieldElement {
       } else {
         // Validate collections
         if (type.isRealmCollection) {
-          if (!isFinal) {
```

Comment:
These changes are part of master from #253 Could you please remove these files changes from this PR. So all generator files changes here

Reply:
Done. Merged from `master`.